api/agent.go

```diff
@@ -162,7 +162,7 @@ const (
 	// MemberTagKeyReadReplica is the key used to indicate that the member is a read
 	// replica server (will remain a Raft non-voter).
 	// Read Replicas are a Consul Enterprise feature.
-	MemberTagKeyReadReplica = "nonvoter"
+	MemberTagKeyReadReplica = "read_replica"
```

Comment:
This had never gone out in a release yet which is why this isn't a breaking change.